balboa-client-dispatcher/src/main/scala/com/socrata/balboa/config/DispatcherConfig.scala

```diff
@@ -0,0 +1,33 @@
+package com.socrata.balboa.config
+
+import com.socrata.balboa.config.ClientType.ClientType
+import com.socrata.balboa.metrics.config.{Configuration, Keys}
+import org.slf4j.LoggerFactory
+
+/**
+ * All inclusive Configuration file
+ */
+object DispatcherConfig {
+
+  private val Log = LoggerFactory.getLogger("DispatcherConfig")
+
+  /**
+   * @return Client Types parsed from configuration.
+   */
+  def clientTypes: Array[ClientType] = {
+    val clientStrings: Array[String] = Configuration.get().getString(Keys.DISPATCHER_CLIENT_TYPES).split(',')
+
+    // Notify that we found an invalid Client Type with a graceful log message
+    clientStrings.filterNot(s => isClientType(s)) match {
+      case a: Array[String] if a.nonEmpty => Log.warn(s"$a not a valid client type, acceptable client " +
+        s"types ${ClientType.values}")
+      case _ =>
```

Comment:
Add logging. No silent failures. If we fall into the default case, something is probably wrong with our config file.